pkgs/by-name/so/soundalike/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitea,
+  chromaprint,
+  makeWrapper,
+}:
+
+buildGoModule rec {
+  pname = "soundalike";
+  version = "0.1.2";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "derat";
+    repo = "soundalike";
+    rev = "v${version}";
```

You missed a small thing from my earlier review:

```suggestion
    tag = "v${version}";
```

---

good catch!

rebased and updated here: https://github.com/NixOS/nixpkgs/pull/388332/commits/3e3e5c9e2ec7d227ff0e6e499fb4ada994d1c42b